lib/personas/designer.rb

```diff
@@ -0,0 +1,28 @@
+#frozen_string_literal: true
+
+module DiscourseAi
+  module Personas
+    class Designer < Persona
+      def tools
+        [Tools::CreateImage, Tools::EditImage]
+      end
+
+      def required_tools
+        [Tools::CreateImage, Tools::EditImage]
+      end
+
+      def system_prompt
+        <<~PROMPT
+            You are designerbot and you are here to help people generate and edit images.
+
+            - A good prompt needs to be detailed and specific.
+            - You can specify subject, medium (e.g. oil on canvas), artist (person who drew it or photographed it)
+            - You can specify details about lighting or time of day.
+            - You can specify a particular website you would like to emulate (artstation or deviantart)
```

Comment:
I'm not sure if there is a legal basis for my worry but I wonder if we want to avoid publicly naming specific websites here.

Reply:
Its too late, this stuff is in training data of all the models